### PR TITLE
[SAT-48] Bootstrap operations.

### PR DESCRIPTION
**Pull request by rozenbom**
## Description of changes in release / Impact of release:
Bootstrap operations.

### Is this a breaking change?
  - [ ] Yes
  - [x] No

### If you answered Yes then describe why is it so
(insert text here)

#### Additional details go here
(insert text here)

### CODE REVIEW

File: satellite/vault/vault_handler.py [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
-from typing import Tuple
-
 from mitmproxy.connections import ServerConnection
 from mitmproxy.http import HTTPFlow
 
 from satellite import ctx
+from satellite import audit_logs
 from satellite.db.models.route import Phase
-from satellite.proxy import audit_logs
+from satellite.fco.pipeline import build_pipeline
```
Comment (mahmoudimus) [inline satellite/vault/vault_handler.py:9]: Can we not use the word `fco` publicly. Can you call it `operations`?

**Comment by mahmoudimus** [comment]
Holy moly. This is super amazing.

File: satellite/vault/vault_handler.py [hunk 2]
```diff
@@ -50,56 +42,57 @@ def response(self, flow: HTTPFlow):
...
-                {'id': rule.id, 'operation_applied': op_applied}
-                for rule, op_applied in zip(route_filters, ops_applications)
-            ]
-            return content, {'route_id': route.id, 'filters': matched_filters}
+        if not route:
+            return
+
+        match_details = {'filters': [], 'route_id': route.id}
+        filters = match_details['filters']
+        for rule_entry in rule_entries:
```
Comment (mjallday) [inline satellite/vault/vault_handler.py:85]: routes and filters. do not use the terms rule chains or entries.

File: satellite/tests/vault/test_valut_handler.py [hunk 3]
```diff
@@ -2,15 +2,21 @@
```
Comment (mjallday) [inline satellite/tests/vault/test_valut_handler.py:2]: file is misspelled.

File: satellite/vault/vault_handler.py (hunk 1 above)
Comment (rozenbom) [reply satellite/vault/vault_handler.py]: Done

File: satellite/vault/vault_handler.py (hunk 2 above)
Comment (rozenbom) [reply satellite/vault/vault_handler.py]: Will use routes and filters from now on but we have rule chains/entries all over the code. If we want the total renaming I'd ticketed/implemented it separately.

File: satellite/tests/vault/test_valut_handler.py (hunk 3 above)
Comment (rozenbom) [reply satellite/tests/vault/test_valut_handler.py:2]: Thank you for spotting it.

File: satellite/audit_logs/records.py [hunk 4]
```diff
@@ -83,33 +85,23 @@ class VaultTrafficLogRecord(AuditLogRecord):
...
-
-
-def emit(record: AuditLogRecord):
-    _sig_audit_log.send(record=record)
-
-
-def subscribe(callback: Callable):
-    _sig_audit_log.connect(lambda _, record: callback(record), weak=False)
+@dataclass
+class OperationPipelineEvaluationLogRecord(AuditLogRecord):
```
Comment (aslepakurov) [inline satellite/audit_logs/records.py:89]: Let's not use `OperationPipeline...` in vgs satellite. Let's use `OperationEvaluationLogRecord`

**Review by aslepakurov — APPROVED** [review]

**Pull request merged**